plugin.xml

```diff
@@ -31,7 +31,7 @@
 
   <engines>
     <engine name="cordova-android" version="6.1.2" />
```

Comment:
I tried it. It's broken right now. Spoke to their lead dev - he said `6.2.3` is gonna drop soon with fixes for that. I'll wait and then upgrade to `6.2.3` directly.